config/env/development.js

```diff
@@ -13,5 +13,9 @@ module.exports = {
 
    // If you want to debug regression tests, you will need the following which is also in the test config:
    zapHostName: "192.168.56.20",
-   zapPort: "8080"
+   zapPort: "8080",
```

Comment:
Do we need ZAP specific properties in dev config file? I noticed we have it as part of the test config as well.